### PR TITLE
Adding Bizible mixins for B2B Account/Person/Opportunity

### PR DESCRIPTION
**Pull request by mjasrotia**
Please link to the issue #…

### CODE REVIEW

File: extensions/adobe/experience/bizible/bizible-b2b-person-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          "title": "Contact/Lead Stage",
+          "description": "Current stage of a business Contact or Lead.",
+          "type": "string"
+        },
+        "bizible:stagePrevious": {
+          "title": "Contact/Lead Previous Stage",
+          "description": "Previous stage of a business Contact or Lead.",
+          "type": "string"
+        },
+        "bizible:personSource": {
```
Comment (peichiang) [inline extensions/adobe/experience/bizible/bizible-b2b-person-details.schema.json:33]: Should we use the existing generic personSource?

Comment (mjasrotia) [reply extensions/adobe/experience/bizible/bizible-b2b-person-details.schema.json]: Resolved

**Comment by peichiang** [comment]
reviewed and accepted.

File: extensions/adobe/experience/bizible/bizible-account-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/bizible-account-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Bizible Account Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/account"],
+  "meta:extends": [
```
Comment (cmathis) [inline extensions/adobe/experience/bizible/bizible-account-details.schema.json:15]: Not sure we should do this. Instead this mixin can just introduce two new fields and be included with the other two mixins. The goal is not to create a mixin that replaces the others.

**Review by cmathis — APPROVED** [review]

**Pull request merged**